ui/components/app/confirmations/alerts/alert-modal/index.scss

```diff
@@ -0,0 +1,14 @@
+@use "design-system";
+
+.alert-modal {
+  &__alert-details {
+    list-style-type: disc;
+    word-wrap: break-word;
+  }
+
+  &__acknowledge-checkbox {
+    @include design-system.H6;
+
+    padding-top: 2px;
+  }
```

Comment:
```suggestion
    input.mm-checkbox__input:not(.mm-checkbox__input--checked) {
      background-color: var(--brand-colors-white-white000);
    }
    }
```

This will make the checkbox background white if it's not selected.

Reply:
Before
![Screenshot 2024-04-26 at 12 30 01](https://github.com/MetaMask/metamask-extension/assets/7644512/517c53de-8f91-4f2d-b705-3e4b7c20378f)

After 
![Screenshot 2024-04-26 at 12 29 45](https://github.com/MetaMask/metamask-extension/assets/7644512/f0e6ef42-a5b6-40ce-bd8f-a44b0d58f17b)

Reply:
Hey @OGPoyraz and @vinistevam, I checked the Figma component and it is supposed to have a white background. We can merge this PR and remove this code https://github.com/MetaMask/metamask-extension/pull/24262

Reply:
I am resolving this since due to @georgewrmarshall 's comment